jenkins-webhook/1.0.0/README.md

```diff
@@ -0,0 +1,120 @@
+The task will look like this:
+
+![jenkins async](https://raw.githubusercontent.com/keptn-contrib/artifacthub/main/jenkins-webhook/1.0.0/assets/async-task.jpg)
+
+## Synchronously Trigger Jenkins
+
+In the bridge, navigate to a project and go to the Settings > Integrations page.
+
+Select the webhook service and click the `+ Add subscription` button.
+
+When configuring the webhook, set `Send started event` to `automatically`. Set `Send finished event` to `by webhook receiver`.
+
+![jenkins sync](https://raw.githubusercontent.com/keptn-contrib/artifacthub/main/jenkins-webhook/1.0.0/assets/sync-started-finished.jpg)
+
+With these settings, the webhook service will automatically generate and send a `taskName.started` event back to Keptn when the webhook is fired. Before progressing, Keptn will then wait for a corresponding `taskName.finished` task which must be generated by Jenkins and sent from Jenkins back to Keptn.
+
+Should Jenkins fail to send the `taskName.finished` event, the task will never end.
+
+1. Store the Keptn API token as a Jenkins credential
+2. Parameterise the Jenkins build
+3. Trigger the job using `/buildWithParameters` endpoint
+4. On triggering a job, send important Keptn parameters to Jenkins as Job parameters (see below)
+5. (optional) Use build parameters sent above to send `taskName.status.changed` events during execution
+6. Use build parameters sent above to craft a `taskName.finished` event and send to Keptn
+
+### Jenkins Build Parameters
+Parameterise the Jenkins job and add the following parameters:
+
+1. `keptnProject`, `keptnService` and `keptnStage`
+2. `keptnContext` and `keptnId`
+3. `keptnTask`
+
+### Craft Webhook Requests
+
+The URL should be a POST and look something like this:
+
+```
+https://myJenkins.com/job/JobName/buildWithParameters?keptnProject={{.data.project}}&keptnService={{.data.service}}&keptnStage={{.data.stage}}&keptnContext={{.shkeptncontext}}&keptnId={{.id}}&keptnTask=<YourTaskName>
+```
+
+![sync webhook](https://raw.githubusercontent.com/keptn-contrib/artifacthub/main/jenkins-webhook/1.0.0/assets/sync-webhook.jpg)
+
+Items in double curly brackets are variables which will be pulled from the `taskName.triggered` CloudEvent and replaced at runtime.
+
+These parameters are required so you can build a `taskName.finished` event when the job has finished.
+
+Sending one or more `taskName.status.changed` event(s) **during** execution is also possible. Adjust the example below so the `type` is `sh.keptn.event.<taskName>.status.changed` rather than `sh.keptn.event.<taskName>.finished`.
+
+### Sample Pipeline
+
+```
+pipeline {
+    agent any
+
+    stages {
+        stage('Hello') {
+            steps {
+                
+                echo 'Hello World'
+                sleep 10
+                
+                withCredentials([string(credentialsId: 'keptn-api-token', variable: 'KEPTN_API_TOKEN')]) {
+                  sh """
+                    curl -X POST http://myKeptn.com/api/v1/event \
```

Comment:
Any chance we can put the KEPTN URL in a variable too? SAme as KEPTN_API_TOKEN? In [Keptn jenkins Library](https://github.com/keptn-sandbox/keptn-jenkins-library#usage) we used an approach where we stored all of it either as a secret or environment variable (I think secret would be better). The variables would then be available in your code as

* `KEPTN_ENDPOINT`
* `KEPTN_BRIDGE` (probably not needed right now)
* `KEPTN_API_TOKEN` - you already have that :+1: